### PR TITLE
Ecommerce AEO program

### PR DESCRIPTION
**Pull request by nissimsan**
This PR updates schemas for an AEO program specifically for the new supply chain actors:
- Online Sellers
- Marketplaces
- Fulfillment Centers

At high level, there are three types of credentials in play: 
1. Organization Credentials: a) Marketplace know-your-seller, b) Seller AEO, c) Marketplace AEO, d) Fulfillment AEO
2. Product Registration Credentials: a) Seller Product listing, b) Fulfillment know-your-inventory, c) IPR 
3. Transaction Registration Credentials a) Order Registration, b) Fulfillment confirmation

Together, these enable linking shipments/transactions back to pre-registered product documentation submitted by program participants; with import pre-approval and AEO certificates as the main incentive mechanisms. 

The PR replaces some previous done Ecommerce schemas aiming for the same goals, but now aligns to our since established modelling conventions.

### CODE REVIEW

File: docs/openapi/components/schemas/common/BankAccountHolderVerification.yml
Comment (OR13) [inline docs/openapi/components/schemas/common/BankAccountHolderVerification.yml:15]: I really don't like this name...

File: docs/openapi/components/schemas/common/BankAccountHolderVerification.yml
Comment (OR13) [inline docs/openapi/components/schemas/common/BankAccountHolderVerification.yml:14]: I wonder if we can use another term for this.

**Review by OR13 — APPROVED** [review]
File issues for the comments, I don't feel we need to hold up the PR for them as long as they are tracked.

File: docs/openapi/components/schemas/common/BankAccountHolderVerification.yml
Comment (nissimsan) [reply docs/openapi/components/schemas/common/BankAccountHolderVerification.yml:15]: Ahh, "VerificationMethod"... Yea, that has other connotations. I'll find a better name...

File: docs/openapi/components/schemas/common/BankAccountHolderVerification.yml
Comment (nissimsan) [reply docs/openapi/components/schemas/common/BankAccountHolderVerification.yml:14]: Maybe "check" or "validate"?

**Comment by BenjaminMoe** [comment]
Looks like jest is failing:
```
  ● Verifiable Credentials › ProductRegistrationCredential

    expect(received).toBe(expected) // Object.is equality

    Expected: true
    Received: false

      42 |         checkStatus: () => ({ verified: true }),
      43 |       });
    > 44 |       expect(verified).toBe(true);
         |                        ^
      45 |     });
      46 |   });
      47 | });
      ```

Run `npm i` to recreate the context, then run the regenerate script to resign the credential.

**Review by BenjaminMoe — APPROVED** [review]
Approving, assuming CI is fixed.

File: docs/openapi/components/schemas/common/BankAccountHolderVerification.yml
Comment (nissimsan) [reply docs/openapi/components/schemas/common/BankAccountHolderVerification.yml]: Updated.

**Review by mkhraisha — APPROVED** [review]

**Comment by nissimsan** [comment]
Closes https://github.com/w3c-ccg/traceability-vocab/issues/564

**Pull request merged**
